lib/mbaasRequest/mbaasRequest.js

```diff
@@ -279,6 +279,7 @@ function doFHMbaaSRequest(params, cb) {
   } else {
     //Normal call.
     log.logger.debug("FH-MBAAS-CLIENT: doFHMbaaSRequest Normal Request");
+    console.log("--------making a normal call: ", params);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)